extensions/tailscale/src/status.tsx

```diff
@@ -0,0 +1,23 @@
+import { updateCommandMetadata } from "@raycast/api";
+import { getDevices, getStatus } from "./shared";
+
+export default async function Status() {
+  let subtitle: string;
+  try {
+    const data = getStatus();
+
+    // tailscale is guaranteed to be online because getStats throws if it isn't
+
+    const magicDNSSuffix = data.MagicDNSSuffix;
+    const devices = getDevices(data);
+    const activeExitNode = devices.find((d) => d.exitnode);
+
+    subtitle = `✅ Tailscale is connected on ${magicDNSSuffix}`;
```

Comment:
I think this can just be `Connected on ${magicDNSSuffix}` because Tailscale is already in the command name.

![Screenshot 2023-11-10 at 11 46 46@2x](https://github.com/raycast/extensions/assets/303731/d2b27141-e922-4f2e-9122-1a155687f211)